cranelift/filetests/filetests/licm/rewrite-jump-table.clif

```diff
@@ -0,0 +1,28 @@
+test licm
+target x86_64
+
+function %rewrite_jump_table() {
+    jt0 = jump_table [block1, block2]
+
+                        block0:
+                            v0 = iconst.i64 1
+                            v1 = jump_table_base.i64 jt0
+                            v2 = jump_table_entry.i64 v0, v1, 4, jt0
+                            v3 = iadd v1, v2
+[RexOp1indirect_jmp#40ff]   indirect_jump_table_br v3, jt0
```

Comment:
Can we remove the encodings here? They'll cause issues when we switch to the new backend by default and I don't think they're needed for the test.

Reply:
We can remove the encodings iff the test runs with the new backend, otherwise `Verifier::verify_encodings` fails because  some `jump` insts with encodings is inserted here.
https://github.com/bytecodealliance/wasmtime/blob/2cec20aa57037ecdfaef46176f31fe8f6b33c2d8/cranelift/codegen/src/licm.rs#L94-L98
So, I'm inclined to remove all encodings from the test and change the target machine to `aarch64` from `x86-64`,  what do you think about it?

Reply:
Yes, that sounds good, thanks!